tests/settings.py

```diff
@@ -115,3 +115,4 @@
 AI_ASSISTANT_CAN_UPDATE_MESSAGE_FN = "django_ai_assistant.permissions.owns_thread"
 AI_ASSISTANT_CAN_DELETE_MESSAGE_FN = "django_ai_assistant.permissions.owns_thread"
 AI_ASSISTANT_CAN_RUN_ASSISTANT = "django_ai_assistant.permissions.allow_all"
+AI_ASSISTANT_PRIMARY_KEY_FIELD = "uuid"  # Options: 'auto', 'uuid', 'string'
```

Comment:
Leave auto for tests. But I think we don't actually need this if we do https://github.com/vintasoftware/django-ai-assistant/pull/110/files#r1648900685

Reply:
Yeah, I was testing things with UUID, but the default would be auto.